tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -30,6 +30,7 @@ limitations under the License.
 #include "absl/strings/str_cat.h"
 #include "absl/strings/string_view.h"
 #include "tensorflow/compiler/tf2tensorrt/convert/utils.h"
+#include "tensorflow/compiler/tf2tensorrt/utils/py_utils.h"
```

Comment:
Please avoid depending on this if possible. This is for python only, it depends on TF's dso_loader, and I remembered it has caused some problems before (though I forgot how exactly).

Instead, please add new methods Get{Linked,Loaded}TensorRTVersion and maybe let them return a string to make it more convenient. We already duplicate the code in line ~1189.

Thanks.

Reply:
Done. Could you please review if I added the headerfiles and namespaces properly?